src/index.html

```diff
@@ -6,9 +6,92 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navbar">
+      <a
+      class="link__logo"
+       href="#home"
+      >
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+            href="#"
+            class="nav__link is-active"
+            >
+            apple
+            </a>
```

Comment:
It is realy hard to read when everything is aligned the same level
```suggestion
            <a
              href="#"
              class="nav__link is-active"
            >
              apple
            </a>
```